.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -96,6 +101,7 @@ jobs:
       PREVIOUS_KAYOBE_IMAGE: ghcr.io/stackhpc/stackhpc-kayobe-config:stackhpc-2023.1
       # NOTE(upgrade): Reference the PREVIOUS release branch here.
       PREVIOUS_BRANCH: stackhpc/2023.1
+      
```

Comment:
None of the other workflows have whitespace here, could you remove it?
```suggestion
```
And no, we shouldn't add whitespace to all the other workflows